afl/src/lib.rs

```diff
@@ -68,10 +70,15 @@ where
 
     let mut input = vec![];
 
+    let loop_count: u32 = env::var("AFL_FUZZER_LOOPCOUNT")
+        .unwrap_or_else(|_| "4294967295".to_string())
```

Comment:
I think it would be better to panic when `AFL_FUZZER_LOOPCOUNT` doesn't parse as a natural number.

Reply:
Also, when the environment variable is not set, it would probably be better to use the maximum integer value directly, rather than parse its string representation.

Reply:
fixed the usize and error on parse. my rust skills are not good enough to know how to fix the "not need to parse" part :)